rvm-test/Gemfile

```diff
@@ -1,3 +0,0 @@
-source 'https://rubygems.org'
-
-gem 'tf'
```

Comment:
Why would you remove this file?

Reply:
Unintentional deletion. Restored and branch updated.